fix(#1529): Pagination - component breaks if you change items per page

syedszeeshan opened the pull request.

lizhuomeng71 commented:
issue is verified on react and angular

chrisolsen reviewed (changes requested):
You will need to squash your commits into one. 

Can you also reword your commit to something like `fix loss of state on items per page page change'

syedszeeshan commented:
> You will need to squash your commits into one.
> 
> Can you also reword your commit to something like `fix loss of state on items per page page change`

Changed to a single commit and updated commit message.
